### ISSUE TITLE
Ocsigenserver 2.9 breaks Eliom 6.2 due to lost implied Lwt constraint

### ISSUE THREAD

**Issue by paurkedal** [ocaml/opam-repository#11348]
After #11340, Eliom 6.2.0 fails to build since Lwt is upgrade to 3.2.1 since Ocsigenserver no longer has the Lwt version constraint which Eliom depended on due to a reference to `lwt.react`.  I suggest adding an explicit dependency and constraint for lwt in the eliom package itself.  (Though it would be nice to have Eliom with the most recent Lwt if possible.)

**Comment by jpdeplaix** [comment]
Sorry for that. The reverse dependency checks are checking using OCaml 4.05.0 and Eliom 6.2.0 is not installable with that version of OCaml.

If you are willing to contribute by sending a PR, you can add the following constraint to the eliom.6.2 package:
```
"lwt" {>= "2.5.0" & < "3.0.0"}
```
Otherwise I'll do it later

**Comment by paurkedal** [comment]
Interesting, I wasn't sure how testing worked for this case. At least this shouldn't be a common type of miss, then. I'll prepare a PR.
